modules/express/test/unit/clientRoutes/signPayload.ts

```diff
@@ -69,3 +68,63 @@ describe('Sign an arbitrary payload with trading account key', function () {
     await handleV2OFCSignPayload(req).should.be.resolvedWith(expectedResponse);
   });
 });
+
+describe('Sign an arbitrary payload with trading account key read from the local file system (external signing mode)', () => {
+  let bitgo: BitGo;
+
+  const walletId = '61f039aad587c2000745c687373e0fa9';
+  const walletPassword = 'wDX058%c4plL1@pP';
+  const secret =
+    'xprv9s21ZrQH143K3EuPWCBuqnWxydaQV6et9htQige4EswvcHKEzNmkVmwTwKoadyHzJYppuADB7Us7AbaNLToNvoFoSxuWqndQRYtnNy5DUY2';
+  const validPrv =
+  '{"61f039aad587c2000745c687373e0fa9":"{\\"iv\\":\\"+1u1Y9cvsYuRMeyH2slnXQ==\\",\\"v\\":1,\\"iter\\":10000,\\"ks\\":256,\\"ts\\":64,\\"mode\\":\\"ccm\\",\\"adata\\":\\"\\",\\"cipher\\":\\"aes\\",\\"salt\\":\\"54kOXTqJ9mc=\\",\\"ct\\":\\"JF5wQ82wa1dYyFxFlbHCvK4a+A6MTHdhOqc5uXsz2icWhkY2Lin/3Ab8ZwvwDaR1JYKmC/g1gXIGwVZEOl1M/bRHY420h7sDtmTS6Ebse5NWbF0ItfUJlk6HVATGa+C6mkbaVxJ4kQW/ehnT3riqzU069ATPz8E=\\"}"}';
+
+  const payload = {
+    this: {
+      is: {
+        a: 'payload',
+      },
+    },
+  };
+
+  before(() => {
+    bitgo = new BitGo({ env: 'test' });
+  });
+
+  it('should work', async () => {
```

Comment:
Please have a more descriptive test name.

Reply:
ah I meant to update this. thanks!